examples/stream.py

```diff
@@ -0,0 +1,18 @@
+"""Demonstrates musekafka.consumer.App in stream mode.
+
+To use, first start up the Kafka cluster using `make start-dev-kafka-cluster`.
+Next, start this example by running `./devenv python examples/stream.py`.
+Finally, use a tool such as kafkacat (https://github.com/edenhill/kafkacat) to send
+messages to the broker.
+"""
+
+import muselog
+
+from musekafka import consumers
+
+muselog.setup_logging(module_log_levels={"musekafka": "DEBUG"})
+
+app = consumers.App("stream_example", ["localhost:9092"], ["stream_example"])
```

Comment:
How would you be able to stream or batch multiple topics? I couldn't really understand that from looking at the consumer interface?

Reply:
I know it takes in a list of topics but how would I be able to distinguish between the two topics? Would it be a single for loop and you have to check the topic on the message and it can come in any order?

Reply:
Yes. You would need to check the topic. We don't really do multiple topics per single consumer instance at the moment. This will need to be improved upon if we need to do that, especially as older versions of Confluent Schema Registry will do not support multiple schemata per topic.

Reply:
Gotcha ok I see. If its not well supported and not needed atm, should we even expose it in the interface yet?